x-pack/plugins/security_solution/public/common/components/navigation/solution_grouped_nav/solution_grouped_nav_item.tsx

```diff
@@ -0,0 +1,255 @@
+        {
+          label: 'ENDPOINTS',
+          itemIds: [
+            SecurityPageName.endpoints,
+            SecurityPageName.policies,
+            SecurityPageName.trustedApps,
+            SecurityPageName.eventFilters,
+            SecurityPageName.blocklist,
+            SecurityPageName.hostIsolationExceptions,
+          ],
+        },
+      ],
+      items: [
+        {
+          id: SecurityPageName.rules,
+          label: 'Rules',
+          description: 'The description here',
+          ...getSecuritySolutionLinkProps({ deepLinkId: SecurityPageName.rules }),
+        },
+        {
+          id: SecurityPageName.exceptions,
+          label: 'Exceptions',
+          description: 'The description goes here',
+          ...getSecuritySolutionLinkProps({ deepLinkId: SecurityPageName.exceptions }),
+        },
+        {
+          id: SecurityPageName.endpoints,
+          label: 'Endpoints',
+          description: 'The description goes here',
+          ...getSecuritySolutionLinkProps({ deepLinkId: SecurityPageName.endpoints }),
+        },
+        {
+          id: SecurityPageName.policies,
+          label: 'Policies',
+          description: 'The description goes here',
+          ...getSecuritySolutionLinkProps({ deepLinkId: SecurityPageName.policies }),
+        },
+        {
+          id: SecurityPageName.trustedApps,
+          label: 'Trusted applications',
+          description: 'The description goes here',
+          ...getSecuritySolutionLinkProps({ deepLinkId: SecurityPageName.trustedApps }),
+        },
+        {
+          id: SecurityPageName.eventFilters,
+          label: 'Event filters',
+          description: 'The description goes here',
+          ...getSecuritySolutionLinkProps({ deepLinkId: SecurityPageName.eventFilters }),
+        },
+        {
+          id: SecurityPageName.blocklist,
+          label: 'Blocklist',
+          description: 'The description goes here',
+          ...getSecuritySolutionLinkProps({ deepLinkId: SecurityPageName.blocklist }),
+        },
+        {
+          id: SecurityPageName.hostIsolationExceptions,
+          label: 'Host Isolation IP exceptions',
+          description: 'The description goes here',
+          ...getSecuritySolutionLinkProps({ deepLinkId: SecurityPageName.hostIsolationExceptions }),
+        },
+      ],
+    },
+  ];
+};
+
+export const useFooterNavItems: () => NavItem[] = () => {
+  const { getAppUrl, navigateTo } = useNavigation();
+
+  const getOnClick =
+    (deepLinkId: string): React.MouseEventHandler =>
+    (ev) => {
+      ev.preventDefault();
+      navigateTo({ deepLinkId });
+    };
+
+  return [
+    {
+      id: 'get_started',
+      render: () => (
+        <EuiLink href="/get_started">
+          <EuiListGroupItem
+            label=" GET STARTED"
+            size="xs"
+            color="text"
+            icon={
+              <span role="img" aria-label="get started">
+                {'🚀 '}
```

Comment:
We've never used emoji's in our UI. Will need to check on if this is ok.

Reply:
yep, I was not sure either

Reply:
I confirmed with other design leads that we should not be using emoji's. If we don't have an icon that expresses the same meaning, we'll need to design one that adheres to the rest of our icon styles. Can you ask the designer to either pick an existing icon or design one specifically for this case?

Reply:
yes, thanks for checking. I'll remove it by now. 
cc @bfishel